### DISCUSSION TITLE
ETA for 4.0

### DISCUSSION THREAD

**Discussion by Rmarian** [rabbitmq/rabbitmq-server#9711] [Questions] [answered]
Hi @michaelklishin,

Do you know if there is a rough ETA for RabbitMQ 4.0 release? We are interested in the new improved Raft based partition handling provided in this version with Khepri, that is why we ask.

Thanks,
Radu.

**Comment by michaelklishin** [comment, marked as answer]
Our team does not provide ETAs. Khepri will ship behind a feature flag in 3.13.0 later this year. 4.0 is not expected this year.
